### ISSUE TITLE
CR: "follows ... conventions ... [JSON-LD]" sounds too much like it requires JSON-LD - suggest rewording

### ISSUE THREAD

**Issue by tantek** [jasnell/w3c-socialwg-activitystreams#264]
http://www.w3.org/TR/activitystreams-core/

in 1.2 Serialization Notes, the text:

"follows the conventions defined by the [JSON-LD] specification" 

sounds too much like it requires JSON-LD processing, which we do not want to convey to developers.

I would prefer something like "a subset of JSON that conforms to JSON-LD syntax constraints, but does not require JSON-LD processing".

Specific rewording up to editor's choice.

**Comment by akuckartz** [comment]
:+1: (I think that the use of the word "conventions" was not correct)
